### DISCUSSION TITLE
Uppercase accents

### DISCUSSION THREAD

**Discussion by EMRD95** [monkeytypegame/monkeytype#3722] [Questions]
With some keyboards it's impossible to type an uppercase character with an accent, e.g. typing the character À can be typed only with the numpad (alt+0192).
I tried to add this to a language .json but it doesn't seem to be the way to go.
"accents": [
    ["ĀÁǍÀ", "A"],
    ["ŌÓǑÒ", "O"],
    ["ĒÉĚÈ", "E"],
    ["ĪÍǏÌ", "I"],
    ["ŪÚǓÙ", "U"],
    ["ÜǕǗǙǛ", "V"]
  ],
Any way to add a lazy mode for upper characters with accent for the french language (at least)?

**Comment by EMRD95** [comment]
Solved by installing a special french keyboard layout on Windows.
https://skitiks.fr/#bonus

**Comment by MaximilienGora** [comment]
You can use different keyboard layouts!
